RetsTalk/RetsTalkTests/CoreDataTests/CoreDataPersistableTests.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  CoreDataPersistableTests.swift
+//  RetsTalkTests
+//
+//  Created by MoonGoon on 11/14/24.
+//
+
+import XCTest
+import CoreData
+
+final class CoreDataPersistableTests: XCTestCase {
+    private var coreDataManager: CoreDataPersistable?
+
+    // MARK: Set up and tear down
+
+    // MARK: Test
+
+    func test_add결과_예상과_동일() throws {
+        // given
+        let content: String = "Hello"
```

Comment:
타입 추론을 활용해도 되지 않을까요 ?
```suggestion
        let content = "Hello"
```

다른 부분도 마찬가지 입니다.

Reply:
타입 추론을 통해 Compile 타임이 증가한다고 생각해서 명시적으로 적어줬습니다!
물론 기본적인 String이기 때문에 큰 차이는 없어서 타입 추론을 활용해도 좋을 것 같네요!

Reply:
유의미한 성능차이가 있는 것이 아니라면 가독성을 위해 생략할 수 있는 부분은 최대한 생략하는 것이 더 좋을 것 같습니다.